src/warc_reader.rs

```diff
@@ -54,10 +54,10 @@ impl WarcReader<BufReader<fs::File>> {
     pub fn from_path<P: AsRef<Path>>(path: P) -> io::Result<Self> {
         let file = fs::OpenOptions::new()
             .read(true)
-            .write(true)
             .create(true)
+            .truncate(false)
             .open(&path)?;
-        let reader = BufReader::with_capacity(1 * MB, file);
```

Comment:
I sort of like having this human readability element for these values.

Reply:
I removed the `1 *` because I got clippy lint https://rust-lang.github.io/rust-clippy/master/index.html#identity_op

Reply:
Ah I see. Thanks for the link. Let's keep the linter happy then.